python/cudf/cudf/tests/test_orc.py

```diff
@@ -410,8 +410,15 @@ def test_orc_reader_decimal_type(datadir, orc_file):
     assert_eq(pdf, df)
 
 
-def test_orc_reader_boolean_type(datadir):
-    file_path = datadir / "TestOrcFile.boolean.orc"
+@pytest.mark.parametrize(
+    "orc_file",
+    [
+        "TestOrcFile.boolean_corruption_0.orc",
+        "TestOrcFile.boolean_corruption_1.orc",
```

Comment:
Can we have more descriptive names for these files? These file have a specific feature that exposes the issue, I'd like to  include that information in the file name, to make test failures more informative.